packages/serverpod_client/lib/src/serverpod_client_shared.dart

```diff
@@ -427,6 +429,54 @@ abstract class ServerpodClientShared extends EndpointCaller {
     await _sendControlCommandToStream('auth', {'key': authKey});
   }
 
+  /// Performs the actual call to the server.
+  /// Subclasses implement bespoke logic for e.g. web / io clients.
+  Future<String> callServerEndpointImpl<T>(
+    Uri url, {
```

Comment:
Discussion: Since this isn't the full implementation for calling the server endpoint I think the name might be a bit missleading. Could we think of a better name to reflect that this just handles the request part of calling the server?

Reply:
Agree!